src/utils/interpolate_data.F90

```diff
@@ -0,0 +1,1230 @@
+module interpolate_data
```

Comment:
It's unclear to me what the long-term state of things like lat/lon interpolation will be in CAM-SIMA (e.g. we might end up using external libraries like ESMF to do all the work for us).  Thus I decided to skip reviewing this particular Fortran module. 

That being said, I don't want to us to completely forget about it either (it could certainly use some modifications if we do decide to stick with it long-term).  So when you get the chance could you create an issue in CAM-SIMA reminding us to either clean-it-up or replace it at some point?  Thanks!

Reply:
Thanks, added #307